batch/create-batch-tables-pod.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: v1
```

Comment:
I can't make a comment above. I think you need to change the name of the file `build-batch2-worker-image-startup.sh`